Makefile

```
@@ -0,0 +1,49 @@
+.PHONY: help all clean build lint fmt check-fmt markdownlint \
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)